docs/Customizing_site.rst

```diff
@@ -4143,8 +4143,11 @@ Customizing a shared layout
 
 In eXo Platform, the top navigation bar is a special container which is
 composed of portlets. All sites share the same top navigation bar that
-is defined in
-``platform-extension.war!/WEB-INF/conf/portal/portal/sharedlayout.xml``.
+is defined in:
```

Comment:
The sentence is confusing since now all sites do not share the same top navigation. We should rephrase it IMO.

Reply:
done